homeassistant/components/fritz/services.py

```diff
@@ -0,0 +1,68 @@
+"""Services for Fritz integration."""
+import logging
+
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.device_registry import async_get as async_get_dev_reg
+from homeassistant.helpers.entity_registry import async_get as async_get_ent_reg
+
+from .const import DOMAIN, FRITZ_SERVICES, SERVICE_REBOOT, SERVICE_RECONNECT
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_services(hass):
+    """Set up services for Fritz integration."""
+    if hass.data.get(FRITZ_SERVICES, False):
+        return
+
+    hass.data[FRITZ_SERVICES] = True
+
+    async def async_call_fritz_service(service_call):
+        """Call correct Fritz service."""
+        fritz_tools = await _async_get_configured_fritz_tools(hass, service_call.data)
```

Comment:
I recently wrote a helper that will get the config entry ids for you:

`from homeassistant.helpers.service import async_extract_config_entry_ids`

`await async_extract_config_entry_ids(hass, call)`